src/TorSharp/Tools/Tor/TorControlClient.cs

```diff
@@ -36,6 +37,16 @@ public async Task CleanCircuitsAsync()
             await SendCommandAsync("SIGNAL NEWNYM", SuccessResponse).ConfigureAwait(false);
         }
 
+        public async Task<String> TrafficReadAsync()
```

Comment:
(I must have lost this comment -- thought it went through)

I think these methods should return an int/long since that is most useful to the caller, I think.

What do you think about returning a class like `TorTraffic` that has two `int`/`long` properties: `Read` and `Written`. Then using the command `GETINFO traffic/read traffic/written` to query for both at the same time?

Reply:
I was thinking to do this but for my purpose was enough. I will see if i can do this next days